### ISSUE TITLE
Ground wetness is on/off

### ISSUE THREAD

**Issue by 900hasse** [900hasse/FS22_REAwheels#2]
Create own ground wetness factor.

**Comment by KCHARRO** [comment]
I was thinking maybe, the way DynamicDirt works, with the water level in the puddles.

Make "Wheels" control if the ground is wet or not in percent values from 1-100. and that DynamicDirt raises/lowers the water level of the puddles starting at 10/12%(change to wet dirt from here) of the Wheels value, and the wheels start to sink a little more and lose some friction (create the table). and find a way to keep the level of humidity/water in the ground, and create a particle system similar to RealMud (which activates at 12% humidity).

In FS19 I made changes to RealMud to be compatible with Seasons, REA and REAdynamicDirt, I had to disable many things in all of them, but the result was great...
